src/org/jetbrains/plugins/scala/compiler/ScalaHydraCompilerConfigurationPanel.scala

```diff
@@ -0,0 +1,81 @@
+package org.jetbrains.plugins.scala.compiler
+
+import java.awt.event.{ActionEvent, FocusEvent, FocusListener}
+import javax.swing.event.DocumentEvent
+
+import com.intellij.openapi.ui.Messages
+import org.jetbrains.plugins.scala.caches.HydraArtifactsCache
+import org.jetbrains.plugins.scala.extensions
+import com.intellij.openapi.project.Project
+import com.intellij.ui.DocumentAdapter
+import org.jetbrains.plugins.scala.project.{ProjectExt, Version, Versions}
+
+import scala.collection.JavaConverters._
+import scala.util.{Failure, Success}
+
+/**
+  * @author Maris Alexandru
+  */
+class ScalaHydraCompilerConfigurationPanel(project: Project, settings: HydraCompilerSettings) extends HydraCompilerConfigurationPanel {
+  private val UnknownVersion = "unknown"
```

Comment:
Is this really need it? Can't we use an `Option` and `None` when the version is `"unknown"`?